plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -85,7 +85,7 @@ def create_optuna_distribution_from_override(override: Override) -> Any:
         if value.shuffle:
             for x in override.sweep_iterator(transformer=Transformer.encode):
                 assert isinstance(
-                    x, (str, int, float)
+                    x, (str, int, float, bool)
                 ), f"A choice sweep expects str, int, or float type. Got {type(x)}."
```

Comment:
Update the message to include bool as well

Reply:
Thanks! I fixed it in https://github.com/facebookresearch/hydra/pull/1132/commits/f45ed55e6e09e01823eae1c318ac3fa746cab5c9.